Sources/Sentry/SentryCrashIntegration.m

```diff
@@ -7,16 +7,24 @@
 //
 
 #if __has_include(<Sentry/Sentry.h>)
-
+#import <UIKit/UIKit.h>
```

Comment:
This is dangerous, this should be in the `#if __has_include(<Sentry/Sentry.h>)`
also; if you build this on Mac this will fail since it's not there, we need to wrap this in 
```
#if SENTRY_HAS_UIKIT
#import <UIKit/UIKit.h>
#endif
```

Reply:
done